### ISSUE TITLE
New pdfminer version 20220319 released

### ISSUE THREAD

**Issue by theraser** [ocrmypdf/OCRmyPDF#933]
**Describe the bug**
python-pdfminer was updated to 20220319, this version isn't supported by OCRmyPDF yet

**To Reproduce**
Update pdfminer to latest version and try to convert any PDF document

**Expected behavior**
No errors

**Screenshots**
```
pkg_resources.DistributionNotFound: The 'pdfminer.six!=20200720,<=20211012,>=20191110' distribution was not found and is required by ocrmypdf
```

**System (please complete the following information):**
 - OS: Arch Linux
 - Python version: 3.10.2
 - OCRmyPDF version: 13.4.1-1

**Installation**
How did you install OCRmyPDF? Did you install it from your operating system's
package manager, or using pip?
All from operating system (Arch Linux)

**Additional context**

**Comment by austinbutler** [comment]
Tests pass and OCR'ing a PDF worked for me after updating the version in `setup.cfg`.

**Comment by jbarlow83** [comment]
Fixed in 04996ca
